src/routes/alerts/alerts.controller.spec.ts

```diff
@@ -1618,7 +1630,7 @@ describe('Alerts (Unit)', () => {
         const alert = alertBuilder().build();
         const timestamp = Date.now().toString();
         const signature = fakeTenderlySignature({
-          signingKey,
+          signingKey: faker.string.nanoid(32),
```

Comment:
Why is this needed? `signingKey` is already being taken from `src/routes/alerts/configuration/__tests__/alerts.configuration.ts` right?

Reply:
It is not because the feature flag for this test is supposed to be false – https://github.com/safe-global/safe-client-gateway/blob/3be66bcdefeb534bec04fbec96acf7397dca4a5d/src/routes/alerts/alerts.controller.spec.ts#L1604

So this test won't have the alerts configuration available.